test-files/cmd/glacial/NetworkConflateCmdInputReverseTest/output.osm

```diff
@@ -5520,8 +5520,8 @@
         <nd ref="-753"/>
```

Comment:
This test output change is due to way joiner and the fact I pulled out previous changes into the new way joiner implementation.  These changes were originally part of PR #2891.